reana/reana_dev/git.py

```diff
@@ -371,8 +376,40 @@ def git_clone(user, component, exclude_components):  # noqa: D301
     if exclude_components:
         exclude_components = exclude_components.split(",")
     components = select_components(component, exclude_components)
+
+    instalation_directory = os.path.realpath(target_directory+'/')
+
+    def yes_no_question():
+        check = str(input("Directory {0} empty. Do you want to clone REANA components here? [y/n] ".format(instalation_directory))).lower().strip()
+        try:
+            if check[0] == 'y':
+                return True
+            elif check[0] == 'n':
+                return False
+            else:
+                return yes_no_question()
+        except Exception as error:
+            return yes_no_question()
+
+        
+    if len(os.listdir(instalation_directory)) == 0:
```

Comment:
Please check whether the directory exists, otherwise the users would get a Python traceback which is not nice:

```console
$ reana-dev git-clone -c reana -u tiborsimko --target-directory /this/does/not/exist
Traceback (most recent call last):
  File "/home/tibor/.virtualenvs/reana/bin/reana-dev", line 8, in <module>
    sys.exit(reana_dev())
  File "/home/tibor/.virtualenvs/reana/lib/python3.8/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/tibor/.virtualenvs/reana/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/tibor/.virtualenvs/reana/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/tibor/.virtualenvs/reana/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/tibor/.virtualenvs/reana/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/tibor/.virtualenvs/reana/lib/python3.8/site-packages/reana/reana_dev/git.py", line 395, in git_clone
    if len(os.listdir(instalation_directory)) == 0:
FileNotFoundError: [Errno 2] No such file or directory: '/this/does/not/exist'
```

You can simply check whether the directory exists, and, if not, reply something like:

```
[ERROR] The desired directory /this/does/not/exist does not exist. Exiting.
```

and then exit with status code 1 to indicate error. (`sys.exit(1)`)
```

Reply:
Done